tests/test_dcrAssembleCoadd.py

```diff
@@ -0,0 +1,295 @@
+        dcrShift = DcrAssembleCoaddTask.dcrShiftCalculate(visitInfo, wcs,
+                                                          self.config.lambdaEff,
+                                                          self.config.filterWidth,
+                                                          self.config.dcrNSubbands)
+        newMaskedImage = DcrAssembleCoaddTask.convolveDcrModelPlane(self.dcrModels[0], dcrShift[0],
+                                                                    bbox=self.bbox,
+                                                                    useFFT=False,
+                                                                    useInverse=False)
+        shift = (dcrShift[0].dy, dcrShift[0].dx)
+        refImage = scipy.ndimage.interpolation.shift(self.dcrModels[0].getImage().getArray(), shift)
+        refVariance = scipy.ndimage.interpolation.shift(self.dcrModels[0].getVariance().getArray(), shift)
+        self.assertFloatsAlmostEqual(newMaskedImage.getImage().getArray(), refImage)
+        self.assertFloatsAlmostEqual(newMaskedImage.getVariance().getArray(), refVariance)
+
+    def testShiftMaskPlane(self):
+        """! Verify the shift calculation for the mask plane.
+
+        The shift of the image and variance planes are tested separately
+        since they are calculated differently.
+        """
+        rotAngle = Angle(0.)
+        azimuth = 200.*afwGeom.degrees
+        elevation = 75.*afwGeom.degrees
+        visitInfo = self.makeDummyVisitInfo(azimuth, elevation)
+        wcs = self.makeDummyWcs(rotAngle, visitInfo=visitInfo)
+        dcrShift = DcrAssembleCoaddTask.dcrShiftCalculate(visitInfo, wcs,
+                                                          self.config.lambdaEff,
+                                                          self.config.filterWidth,
+                                                          self.config.dcrNSubbands)
+        model = self.dcrModels[0]
+        shiftedMask = DcrAssembleCoaddTask.shiftMask(model.getMask(), dcrShift[0],
+                                                     useInverse=False)
+        newMask = DcrAssembleCoaddTask.shiftMask(shiftedMask, dcrShift[0],
+                                                 useInverse=True)
+        detectMask = afwImage.Mask.getPlaneBitMask('DETECTED')
+
+        bufferXSize = np.ceil(np.abs(dcrShift[0].dx)) + 1
+        bufferYSize = np.ceil(np.abs(dcrShift[0].dy)) + 1
+        bboxClip = self.dcrModels[0].getMask().getBBox()
+        bboxClip.grow(afwGeom.Extent2I(-bufferXSize, -bufferYSize))
+
+        # Shifting the mask grows each mask plane by one pixel in the direction of the shift,
+        #  so a shift followed by the reverse shift should be the same as a dilation by one pixel.
+        convolutionStruct = np.array([[True, True, True], [True, True, True], [True, True, True]])
+        maskRefCheck = dilate(model[bboxClip].getMask().getArray() == detectMask,
```

Comment:
Again, why not use our own code (in this instance, `lsst.afw.geom.SpanSet`) instead of a third party's?